packages/caver-utils/src/utils.js

```diff
@@ -710,6 +710,35 @@ const isContractDeployment = (txObject) => {
 
 }
 
+const isValidRole = (role) => {
+  switch(role) {
+    case 'transactionKey':
+    case 'updateKey':
+    case 'feePayerKey':
+      return true
+  }
+  return false
+}
+
+const isDefaultSig = (sig) => {
```

Comment:
The function name is somewhat vague. Can we have another name for this function?

Reply:
Hmmm how about isEmptySig or isEmptySignatures?

Reply:
Yeah, anyone is good to me. :)

Reply:
Function name is changed to isEmptySig :)